infra/main.bicep

```diff
@@ -12,16 +12,23 @@ param location string
 @description('Relative Path of ASA Jar')
 param relativePath string
 
+@allowed([
```

Comment:
don't let developer choose plan, we can set consumption plan by default. If developer want to use basic/standard plan, they can use `azd env set xxx=true` to switch to basic/standard plan.